go/geth_adapter/adapter_test.go

```diff
@@ -548,6 +538,180 @@ func TestRunContextAdapter_Run(t *testing.T) {
 	}
 }
 
+func TestGethAdapter_CorruptValuesReturnErrors(t *testing.T) {
+	tests := map[string]struct {
+		firstBlock  *big.Int
+		baseFee     *big.Int
+		chainID     *big.Int
+		blobBaseFee *big.Int
+		gasPrice    *big.Int
+		difficulty  *big.Int
+	}{
+		"revision": {
+			firstBlock: big.NewInt(1000),
+		},
+		"baseFee": {
+			baseFee: big.NewInt(-1),
```

Comment:
Would a `big.Int` that is equal or larger than 2^256 be detected and correctly handled somewhere?